orchagent/portsorch.cpp

```diff
@@ -2615,6 +2615,15 @@ bool PortsOrch::addPort(const set<int> &lane_set, uint32_t speed, int an, string
 {
     SWSS_LOG_ENTER();
 
+    if (!speed || lane_set.empty())
```

Comment:
Does this impact VS swss initialization?

Reply:
That is why we are deferring until both of them are available. No, it doesn't. cfg used in VS has speed and lane_set defined, so shouldn't be a problem.